resources/audit-policy.yaml

```diff
@@ -95,7 +109,7 @@ rules:
         resources: ["tokenreviews"]
     omitStages:
       - "RequestReceived"
-  # Get responses can be large; skip them.
+  # Get repsonses can be large; skip them.
```

Comment:
`repsones` -> `responses`

Reply:
This while file is pulled from upstream, so I don't want to diverge - makes diffing harder.